File: java/src/bpmlab/exercicio/Exerc06.java
package bpmlab.exercicio;

import java.util.Scanner;

public class Exerc06 {

    public static void main(String[] args) {
        //Calcular e exibir a média aritmética de quatro valores quaisquer que serão digitados.
        int[] valores = new int[4];
        int i = 0;

        System.out.println("insira 4 valores, sendo um por vez");
        while (i < 4) {
            System.out.print(">> ");
            Scanner lerValor = new Scanner(System.in);
            valores[i] = lerValor.nextInt();
            i++;
        }
        float result = 0;
        int a = 0;
        while (a < 4) {
            result += valores[a];
            a++;
        }
        result = result / 4;
        System.out.println("A média aritmética de quatro valores inseridos: " + result);

    }

}

File: python-3.5/Exerc06.py
##Calcular e exibir a média aritmética de quatro valores quaisquer que serão digitados.
valores = []

print("insira 4 valores, sendo um por vez")
for i in range(4):
    print(i)
    valores.append(int(input('>> ')))

result = 0
for i in range(4):
    result += valores[i]
    ##print(valores[i])

print("A média aritmética de quatro valores inseridos: " , result/4)

print("Fim!!!")
